tests/fixtures/fsspec.py

```diff
@@ -0,0 +1,84 @@
+import posixpath
+from pathlib import Path
+
+import fsspec
+import pytest
+from fsspec.implementations.local import AbstractFileSystem, LocalFileSystem, stringify_path
+
+
+class MockFileSystem(AbstractFileSystem):
```

Comment:
I created this mockfs fixture that is not only read but also **write**, this way we can do all our tests based on fsspec using this one :) @albertvillanova